src/index.js

```diff
@@ -14,21 +14,25 @@ import App from "./App";
 import { create } from "jss";
 import { StylesProvider, jssPreset } from "@material-ui/core/styles";
 import compose from "jss-plugin-compose";
+import { loadDevTools } from "./devTools.js";
 
 const jss = create({
     plugins: [...jssPreset().plugins, compose()],
 });
 
-
-ReactDOM.render(
-    <BrowserRouter>
+loadDevTools().then(() => {
```

Comment:
Why is the DevTools bar loaded like this instead of including it inside the App Component?

Reply:
First. it's a matter of semantics. Dev tools are not a part of the application, so they shouldn't be there.

Second, it's one component less to handle when devtools are disabled 

Third, doing it this way, allows this logic (to activate devtools or not) to run on build time, instead of running on clients locally every time they render a page